llap-server/src/java/org/apache/hadoop/hive/llap/daemon/impl/AMReporter.java

```diff
@@ -541,6 +551,14 @@ boolean isDone() {
       return isDone.get();
     }
 
+    void setIsExtCliRequest(boolean val) {
+      isExtCliRequest.set(val);
```

Comment:
nit: for better readability, rename the variable and method to isExternalClientRequest.

Reply:
Sure, I have changed the names to isExternalClientRequest in the recent commit.